### DISCUSSION TITLE
Slider Score Display Position

### DISCUSSION THREAD

**Discussion by Gedminer** [ppy/osu#28148] [Ideas] [answered]
I suggest adding a toggle switch in osu! Lazer! The toggle switch in Lazer controls the display position of slider scores. Players can choose to display scores when the slider movement starts or ends. This customization enhances player choice and contributes to the gaming experience.
------------------
![2024-05-11 15-42-43 mkv_20240511_155235 849](https://github.com/ppy/osu/assets/59013843/e8310140-1bf4-4a59-b5a8-2165de56e949)
osu
-------------------
![2024-05-11 15-49-14 mkv_20240511_155239 330](https://github.com/ppy/osu/assets/59013843/c3fae9a2-3188-4860-9d79-b97a84a2d503)
osu（lazer）

**Comment by bdach** [comment, marked as answer]
This already exists and is called the classic mod.

The placement of the judgement is intentional since it conveys _what_ is being judged. Stable (and classic mod) judge the slider as a whole. lazer judges the slider head specifically.
